docs/service/augmentation.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Container augmentation
+---
+
+The container augmentation provisioning mode allows _extending_ the content of a container image without rebuilding it. Instead, this mechanism modifies a container image during the pull phase made by a Docker-compatible client. Augmented containers are ephemeral: they are not stored in a container repository, and they can only be accessed for a short period of time. The extended content added by Wave is served from the Cloudflare CDN. The augmentation process does perform any _build_ operation behind the scenes.
```

Comment:
I'm not sure if this paragraph is as clear as it could be.  
- Can we be any clearer on how augmentation works?
- What defines the 'short period of time' is it configurable/do we have a default we can mention on Cloud? 
- Is Cloudflare a dependency for self-hosted Wave? If not then how does it work.  
- I'm not sure the last sentence makes sense to me?